react_on_rails_pro/CONTRIBUTING.md

```
@@ -282,14 +282,51 @@ TRACE_REACT_ON_RAILS=true && foreman start -f Procfile.dev
+## Prerequisites
+
+You need authentication for both public and private package registries:
+
+**Public packages (npmjs.org + rubygems.org):**
+- NPM: Run `npm login`
+- RubyGems: Standard credentials via `gem push`
+
+**Private packages (GitHub Packages):**
+- Get a GitHub personal access token with `write:packages` scope
+- Configure `~/.npmrc`:
+  ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)